android/libraries/rib-base/src/main/java/com/badoo/ribs/core/routing/action/DialogRoutingAction.kt

```diff
@@ -0,0 +1,34 @@
+package com.badoo.ribs.core.routing.action
+
+import com.badoo.ribs.core.Node
+import com.badoo.ribs.core.view.RibView
+import com.badoo.ribs.dialog.Dialog
+import com.badoo.ribs.dialog.DialogLauncher
+
+class DialogRoutingAction<V : RibView, Event : Any>(
+    private val dialogLauncher: DialogLauncher,
+    private val dialog: Dialog<Event>
+) : RoutingAction<V> {
+
+    override val allowAttachView: Boolean
+        get() = false
+
+    override fun createRibs(): List<Node<*>> =
+        dialog.createRibs()
+
+    override fun execute() {
+        dialogLauncher.show(dialog)
+    }
+
+    override fun cleanup() {
+        dialogLauncher.hide(dialog)
+    }
+
+    companion object {
+        fun <V : RibView> dialog(
```

Comment:
Can be removed.
Make code not obvious without benefits.

Reply:
See above answer why I think this is a bad idea.